epub33/rs/index.html

```diff
@@ -911,20 +897,10 @@ <h3>Scripting</h3>
 					</li>
 				</ul>
 
-				<div class="note">
-					<p> This specification does not mandate any particular implementation technique for the creation of
-						a unique <a href="https://url.spec.whatwg.org/#origin">origin</a> in the absence of a reliable
-						Web origin (e.g., HTTP URL scheme + host + port). The necessary heuristics may include the
-						combination of the Publication's <a href="https://www.w3.org/TR/epub-33/#dfn-unique-identifier"
-							>Unique Identifier</a> (although, in practice, these identifiers may in fact not guarantee
-						globally-unique identification, that is why it is recommended to combine multiple techniques),
-						filesystem path, <a href="https://www.w3.org/TR/epub-33/#dfn-zip-container">OCF Zip
-							Container</a> checksum, etc. </p>
-					<p> The unicity of the <a href="https://url.spec.whatwg.org/#origin">origin</a> per EPUB Publication
-						instance means that if two different users acquire a copy of the same EPUB Publication, the
-						origins will be different for the two users on those copies even if the same Reading System is
-						used. </p>
-				</div>
+				<p>Note also that, as defined in <a href="#sec-container-iri"></a>, Reading Systems 
+					must assign a unique <a href="https://url.spec.whatwg.org/#origin">origin</a> [[URL]], shared by all <a
+					href="https://www.w3.org/TR/epub-33/#sec-scripted-spine">spine-level scripts</a> of the EPUB Publication. </p>
```

Comment:
I'm not sure that the requirement on the container root URL imply that the unique origin is "shared by all spine-level scripts", or does it? 🤔

This depends on how XHTML Content Documents are processed by RS, and how origins are "assigned" to scripts, both operations I'm not familiar with.

If not, we need to re-add a scripting-specific requirement, or tweak the note.

Reply:
Arrgh... this was probably true with that 4th restrictions that we originally had: if all content in the document had a shared and unique origin then, I suppose, that would have been valid for the spine level scripts. But that is gone so, I guess, it is necessary to reinstate the requirement here. 

Done...